lib/beautiful/init.lua

```diff
@@ -255,7 +256,7 @@ end
 -- @tparam[opt] screen scr Screen to get the DPI information from.
 --   If unspecified, the global font DPI setting will be used.
 function beautiful.get_font_height(name, scr)
-    return load_font(name, beautiful.xresources.get_dpi(scr)).height
+    return load_font(name, beautiful.screen_font_dpi(scr)).height
```

Comment:
Hm, this opens quite a can of worms.
Primary screen changed? Font height changed on all screens!
Primary screen changed DPI value? Font height changed on all screens!
Some random screen changed DPI value? Font height changed for that screen.

I bet that nothing will handle these correctly.